apps/web/components/Forms/ProposalForm.tsx

```diff
@@ -113,7 +113,7 @@ export const ProposalForm = ({
     tokenGarden?.decimals as number,
   );
 
-  const proposalTypeName = proposalTypes[proposalType];
+  const proposalTypeName = poolTypes[proposalType];
```

Comment:
problem here you are accessing a poolTypes array with proposalType index

Reply:
This can be miss leading because proposalType === poolType. We should change this to strategyTypes or change the name of all the variables to poolTypes, because there can't be different types of strategies in the same pool, all of the proposals are of the same type.